### ISSUE TITLE
Release bmm 1.0.0

### ISSUE THREAD

**Issue by GidonFrischkorn** [venpopov/bmm#14]
Additional tasks (added by Ven on May 22):

* [x] Update last figure in readme to reflect addition of priors
* [x] Update NEWS
* [x] fix load of default_prior etc
* [x] Review developer's notes
* [x] Rewrite \{dontrun} examples

First release:

* [x] `usethis::use_news_md()`
* [x] `usethis::use_cran_comments()`
* [x] Update (aspirational) install instructions in README
* [x] Proofread `Title:` and `Description:`
* [x] Check that all exported functions have `@return` and `@examples`
* [x] Check that `Authors@R:` includes a copyright holder (role 'cph')
* [x] Check [licensing of included files](https://r-pkgs.org/license.html#sec-code-you-bundle)
* [x] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [x] `git pull`
* [x] `urlchecker::url_check()`
* [x] `devtools::build_readme()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `git push`
* [x] Draft blog post

Submit to CRAN:

* [x] `usethis::use_version('major')`
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [ ] Add preemptive link to blog post in pkgdown news menu
* [x] `usethis::use_github_release()`
* [x] `usethis::use_dev_version(push = TRUE)`
* [x] `usethis::use_news_md()`
* [x] Finish blog post
* [x] Tweet

**Comment by venpopov** [comment]
@GidonFrischkorn I was wondering if we should submit to CRAN before we resubmit the tutorial paper? That way we can put installation instructions with the regular `install.packages()` function. What do you think? 

I had a look at the issues we have open, and I think the package is in a stable state. Most issues are about new models and functionality, which we could add later.

Perhaps we can do #189, #185, #188  but I think the rest are not needed for us to submit.

**Comment by GidonFrischkorn** [comment]
Agreed. Maybe we can try to have a look at these things until your vacation and also set up the submission to CRAN then.

I do not know how long it takes them to test an initial submission there. But then if any additions are needed, we could address them when you are back and should still be on schedule for having it ready for the resubmission of the revision.

**Comment by venpopov** [comment]
- [ ] Implement the links argument
- [ ] Prerender vignettes and remove example fits (put in a .Rbuildignore folder but keep on github)
- [ ] Do not load brms automatically to speed up loading the package for simulation studies
- [ ] Check and discuss the IMM vignette
- [ ] Categorization scheme

**Comment by venpopov** [comment]
Alright, submitted! 🎉
